build.sbt

```diff
@@ -72,4 +75,4 @@ lazy val scoverageSettings = {
   )
 }
 
-addCommandAlias("runAllChecks", ";clean;compile;coverage;test;it/test;scalastyle;Test/scalastyle;coverageReport")
+addCommandAlias("runAllChecks", ";clean;compile;scalafmtCheckAll;coverage;test;it/test;scalastyle;Test/scalastyle;coverageReport")
```

Comment:
Better to keep scalafmtCheckAll after it/test and before scalastyle to make it consistent with other repos